### PR TITLE
Dynamic Queues

### PR DESCRIPTION
**Pull request by ThrowsException**
#289 
 
Scaffolding out removing the dependencies on hard coded ids in the QueueStore

### CODE REVIEW

**Comment by crsmyz** [comment]
Is this PR still a WIP or is it good to go?

**Comment by ThrowsException** [comment]
@crsmyz I more put this up as a proof of concept about getting rid of the hardcoded Id's to get peoples thoughts on it but if were fine with this approach I can clean this PR up a little bit and remove the WIP tag.

File: frontend/src/views/AllQueues.js [hunk 1]
```diff
@@ -48,48 +49,59 @@ const AllQueues = observer(() => {
...
+        <>
+          <AppBar position="static" color="default">
+            <Tabs
+              value={tabValue}
+              onChange={(event, value) => setTabValue(value)}
+              variant="scrollable"
+              scrollButtons="on"
+              indicatorColor="primary"
+              textColor="primary"
+              aria-label="scrollable force tabs example"
```
Comment (MikeyManoguerra) [inline frontend/src/views/AllQueues.js:82]: I know this was already here but can you make it accurate or remove

Comment (ThrowsException) [reply frontend/src/views/AllQueues.js]: Updated

**Pull request merged**